src/material/button/button.ts

```diff
@@ -97,21 +111,27 @@ export class MatButton extends _MatButtonMixinBase
 
   constructor(elementRef: ElementRef,
               private _focusMonitor: FocusMonitor,
-              @Optional() @Inject(ANIMATION_MODULE_TYPE) public _animationMode: string) {
+              @Optional() @Inject(ANIMATION_MODULE_TYPE) public _animationMode: string,
+              @Optional() @Inject(MAT_BUTTON_DEFAULT_OPTIONS)
+                  defaultOptions?: MatButtonDefaultOptions) {
     super(elementRef);
 
+    const hostElement = this._getHostElement();
+
+    if (!this._hasHostAttributes('type')) {
+      hostElement.type = defaultOptions?.type ?? 'submit';
```

Comment:
Even though the browser defaults it to behave like `submit`, I don't think we should be setting it explicitly in case the default behavior changes.

Reply:
Changed.